components/button-cards/solution-card/solution-card.scss

```diff
@@ -14,11 +14,18 @@
     height: 16px;
     margin-right: 8px;
   }
+  &--orange {
+    background-color: #ec6608;
```

Comment:
Should this colour be added to variables.scss?

Reply:
I also think we should create a separate component for msds-tags, its going to be hard to keep track of different components code nested in other components